lib/events/managerOfferCardEvents.js

```diff
@@ -0,0 +1,7 @@
+if (Meteor.isClient){
+	Template.ManagerCardOffer.events({
+	'click .modSentOfferTrigger': function(){
+		console.log("ASD")
```

Comment:
Mixed double and single quotes.<br>Missing semicolon.<br>'console' is not defined.